src/app/components/ResponsiveLayer/index.tsx

```diff
@@ -36,7 +36,15 @@ export function ResponsiveLayer(props: LayerExtendedProps) {
 
   return (
     <Layer {...props} ref={layerRef} style={{ overflowY: 'auto', ...props.style }}>
-      <Box fill={isMobile}>{props.children}</Box>
+      <Box
+        // Prevents Grommet flex overlap issue in smaller viewport
+        flex={{ shrink: 0 }}
```

Comment:
this doesn't behave the same in landscape
| Before https://mz-settings.oasis-wallet.pages.dev/ | After https://mz-responsivelayerfix.oasis-wallet.pages.dev/ |
| --- | --- |
| ![mz-settings oasis-wallet pages dev_(iPhone SE)](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/e53e0e55-e849-443e-bdc0-136eb0d905bb) | ![mz-responsivelayerfix oasis-wallet pages dev_account_oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km(iPhone SE) (2)](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/976e95b1-2e7c-4874-933e-dd33471b4978) |

Reply:
switched to maxHeight. Seems better now.